expression/bench_test.go

```diff
@@ -195,21 +195,66 @@ func BenchmarkScalarFunctionClone(b *testing.B) {
 	b.ReportAllocs()
 }
 
+// dataGenerator is used to generate data for test.
+type dataGenerator interface {
```

Comment:
Could it be possible to use [`quick.Generator`](https://godoc.org/testing/quick#Generator) instead of creating a new one?

Reply:
Is `dataGenerator's` interface more simple and convenient for our contributor?
`Generate(rand *rand.Rand, size int) reflect.Value` seems weird...